plugins/device/transaction.go

```diff
@@ -7,23 +7,23 @@ import (
 	"github.com/gluster/glusterd2/glusterd2/transaction"
 	deviceapi "github.com/gluster/glusterd2/plugins/device/api"
 
-	"github.com/pborman/uuid"
 	log "github.com/sirupsen/logrus"
 )
 
 func txnPrepareDevice(c transaction.TxnCtx) error {
-	var peerID uuid.UUID
-	var req deviceapi.AddDeviceReq
+	var peerID string
+	var devices []string
```

Comment:
For slightly better readability, you can move declaration of these variables to line just before where they are used i.e just before their respective `c.Get`